shared/admin-tools/elasticsearch/health-migration.js

```diff
@@ -0,0 +1,74 @@
+// arguments: env
+
+/**
+ * for each index, check the index count for alpha and beta
+ *
+ * we might do this for after a migration
+ */
+
+// @path
+const elasticsearchMappings = require('../../../web-api/elasticsearch/elasticsearch-mappings');
+const { getClient } = require('../../../web-api/elasticsearch/client');
+
+const environmentName = process.argv[2] || 'exp1';
+
+console.log(environmentName);
+
+// const getCurrentColor = environmentName => {
+//   DocumentClient.query({
+//     TableName: `efcms-deploy-${environmentName}`,
+
+const getCounts = async ({ indexName, version }) => {
+  const esClient = await getClient({ environmentName, version });
+  const info = await esClient.indices.stats({ index: indexName });
+
+  return info['_all'].total.docs.count;
+};
+
+(async () => {
+  // const currentColor = 'green'; // gets a row from DynamoDB
+
+  // const domains = await listDomains();
+  // console.log(domains);
+
+  const totals = {
+    alpha: 0,
+    beta: 0,
+  };
+
+  console.log(`## ${environmentName} Index Summary`);
+  console.log('+----+----+----+----+');
+  console.log('| Index | Alpha | Beta | Difference |');
+  console.log('+----+----+----+----+');
```

Comment:
Cool.  In a future version, you could even experiment with this: https://developer.mozilla.org/en-US/docs/Web/API/Console/table

Reply:
neat-o. done!